fluss-rpc/src/main/proto/FlussApi.proto

```diff
@@ -955,3 +993,40 @@ message PbDescribeConfig {
   required string config_source = 3;
 }
 
+message PbRebalancePlanForTable {
+  required int64 table_id = 1;
+  repeated PbRebalancePlanForPartition partitions_plan = 2; // for none-partition table, this is empty
+  repeated PbRebalancePlanForBucket buckets_plan = 3; // for partition table, this is empty
+
+}
+
+message PbRebalancePlanForPartition {
+  required int64 partition_id = 1;
+  repeated PbRebalancePlanForBucket buckets_plan = 2;
+}
+
+message PbRebalancePlanForBucket {
+  required int32 bucket_id = 1;
+  optional int32 original_leader = 2;
+  optional int32 new_leader = 3;
+  repeated int32 original_replicas = 4 [packed = true];
+  repeated int32 new_replicas = 5 [packed = true];
+}
+
+message PbRebalanceProcessForTable {
+  required int64 table_id = 1;
+  repeated PbRebalanceProcessForPartition partitions_process = 2;
+  repeated PbRebalanceProcessForBucket buckets_process = 3;
+}
+
+message PbRebalanceProcessForPartition {
+  required int64 partition_id = 1;
+  repeated PbRebalanceProcessForBucket buckets_process = 2;
+}
+
+message PbRebalanceProcessForBucket {
+  required int32 bucket_id = 1;
+  repeated int32 original_replicas = 2 [packed = true];
+  repeated int32 new_replicas = 3 [packed = true];
```

Comment:
Do we need the `optional_leader` and `new_leader`? If need, maybe we can just combine `PbRebalancePlanForBucket` and `rebalance_status` together as the `PbRebalanceProgressForBucket`?